.github/workflows/CI.yml

```diff
@@ -178,6 +165,32 @@ jobs:
                    else
                        packages+=(libgmp-dev libreadline-dev zlib1g-dev)
                    fi
+                   if [[ $TEST_SUITES == *testpackages* ]] ; then
+                       packages+=(
+                         4ti2
+                         libboost-dev
+                         libcdd-dev
+                         libcurl4-openssl-dev
+                         libfplll-dev
+                         libmpc-dev
+                         libmpfi-dev
+                         libmpfr-dev
+                         libncurses5-dev
+                         libzmq3-dev
+                         pari-gp
+                         singular
+                       )
+                   fi
```

Comment:
Maybe we should move this lengthy step into a `dev/ci-install-deps.sh` script?

Reply:
Perhaps. We could also just put it on one line to reduce the number of lines taken.

I worry that putting this stuff in a separate script would make the workflow a bit more opaque. But since we already use several scripts that's probably not important.

For me the ideal thing would be to know why each of these packages is here, or at least, which package(s) require each. 4ti2 and singular are obvious, some of the others less so 🙂,